scripts/quarterlyReport.js

```diff
@@ -0,0 +1,17 @@
+const fs = require('fs');
+const simpleGit = require('simple-git/promise');
+const chalk = require('chalk');
+
+const sGit = simpleGit();
+
+const [, , startDate, endDate] = process.argv;
```

Comment:
The two ',' are a bit confusing. Maybe it is better to write the names of these vars even we are not using them..

Reply:
Yeah, maybe it will be easier to use some command line tool for this